### PR TITLE
Jetpack App: Update App Settings and About screens

### LINKED ISSUE #16197: Jetpack App (Basics): Update About the App
## Description
Update app info under App Settings

 | 
--- | ---

### App Settings
- [x] Update "About WordPress for iOS" to "About Jetpack for iOS"

### About
- [x] Update app icon
- [x] Update Twitter handle
- [x] Update blog name to "https://jetpack.com/blog/"

## Notes

- I don't think we need to update [Terms of Service](https://wordpress.com/tos/) and [Privacy Policy](https://automattic.com/privacy/
) links, since they match what's on https://jetpack.com/
- No need to update the Acknowledgements on the Jetpack app since we don't explicitly mention the app name

### PR DESCRIPTION
**Pull request by momo-ozawa**
Resolves #16197

## Description
This PR updates app specific info under App Setting and About in the Jetpack app
- App Settings
  - Updated "About WordPress for iOS" to "About Jetpack for iOS" 
- About 
  -  Updated app icon (placeholder icon for now) 🚧 
  -  Updated Twitter handle to `@jetpack`
  -  Updated blog url to `https://jetpack.com/blog/`

App Settings | About | Twitter | Blog 
--- | --- | --- | ---
Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 46 01 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 49 42 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 18 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 38
Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 46 07 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 49 49 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 50 26 | Simulator Screen Shot - iPhone 12 Pro - 2021-04-01 at 15 56 35

## How to test

Follow the same steps for both the WordPress and Jetpack target:

1. Go to My Site > Me > App Settings and check the Other section
    - WP: "About WordPress for iOS"
    - Jetpack: "About Jetpack for iOS"
2. Tap on "About ... for iOS"
    - WP: WordPress icon, twitter handle is `@WordPressiOS`, blog is `blog.wordpress.com`
    - Jetpack: "About Jetpack for iOS", twitter handle is `@jetpack`, blog is `jetpack.com`
3. Tap on the Twitter row item
    - WP: Opens the WordPress for iOS twitter
    - Jetpack: Opens the Jetpack twitter
4. Go back the About screen
5. Tap on the Blog row item 
    - WP: Opens the WordPress blog
    - Jetpack: Opens the Jetpack blog

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Tested on both Jetpack and WordPress

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift [hunk 1]
```diff
@@ -464,8 +464,15 @@ private extension AppSettingsViewController {
             action: openApplicationSettings()
         )
 
+        let title: String
+        if AppConfiguration.isJetpack {
+            title = NSLocalizedString("About Jetpack for iOS", comment: "Link to About screen for Jetpack for iOS")
```
Comment (leandroalonso) [inline WordPress/Classes/ViewRelated/Me/App Settings/AppSettingsViewController.swift]: Wdyt about moving those strings to `AppConstants`?

**Review by leandroalonso — CHANGES_REQUESTED** [review]
@momo-ozawa It all works as described! Left one suggestion to remove the `if`.

File: WordPress/Classes/Utility/App Configuration/AppConstants.swift [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+struct AppConstants {
+    static let productTwitterHandle = "@WordPressiOS"
+    static let productTwitterURL = "https://twitter.com/WordPressiOS"
+    static let productBlogURL = "https://blog.wordpress.com"
+}
+
+// MARK: - Localized Strings
+extension AppConstants {
+
+    struct AppSettings {
```
Comment (leandroalonso) [inline WordPress/Classes/Utility/App Configuration/AppConstants.swift]: Perhaps just `Settings`? Then we can refer to it as `AppConstants.Settings.value`

**Review by leandroalonso — APPROVED** [review]
@momo-ozawa Left a small suggestion to change the name of `AppSettings` to `Settings`, but besides that, :shipit:

File: WordPress/Classes/Utility/App Configuration/AppConstants.swift (hunk 2 above)
Comment (momo-ozawa) [reply WordPress/Classes/Utility/App Configuration/AppConstants.swift]: Good point, I think `AppConstants.Settings.value` works 👍

Comment (momo-ozawa) [reply WordPress/Classes/Utility/App Configuration/AppConstants.swift]: Done 15a83a8 😄

**Pull request merged**